BookingQueueSubscriber/BookingQueueSubscriber.Services/NotificationApi/NotificationService.cs

```diff
@@ -1,32 +1,132 @@
+        /// <list type="bullet">
+        /// <item>HearingConfirmationLip</item>
+        /// <item>HearingConfirmationRepresentative</item>
+        /// <item>HearingConfirmationJudge</item>
+        /// <item>HearingConfirmationJoh</item>
+        /// </list>
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participants"></param>
+        /// <returns></returns>
+        Task SendNewSingleDayHearingConfirmationNotification(HearingDto hearing, IEnumerable<ParticipantDto> participants);
+        
+        /// <summary>
+        /// Send the hearing amendment notification for a single hearing
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="originalDateTime"></param>
+        /// <param name="participants"></param>
+        /// <returns></returns>
         Task SendHearingAmendmentNotificationAsync(HearingDto hearing, DateTime originalDateTime, IList<ParticipantDto> participants);
+        
+        /// <summary>
+        /// Send a welcome to VH email to new users. Part of the 1st of 3 new template
+        /// <remarks>NOTE: Do not send to existing users</remarks>
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participant"></param>
+        /// <returns></returns>
         Task SendNewUserWelcomeEmail(HearingDto hearing, ParticipantDto participant);
+        
+        /// <summary>
+        /// Send a single day hearing confirmation to new users. Part of the 2nd of 3 new template
+        /// This is a combination of account details and hearing details
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participant"></param>
+        /// <param name="password"></param>
+        /// <returns></returns>
+        Task SendNewUserSingleDayHearingConfirmationEmail(HearingDto hearing, ParticipantDto participant, string password);
+        
+        /// <summary>
+        /// Send a single day hearing confirmation to existing users. Part of the 2nd of 3 new template
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participant"></param>
+        /// <returns></returns>
+        Task SendExistingUserSingleDayHearingConfirmationEmail(HearingDto hearing, ParticipantDto participant);
+
+        /// <summary>
+        /// Send a multi-day hearing confirmation to new and existing users. Part of the 2nd of 3 new template
+        /// </summary>
+        /// <param name="hearing"></param>
+        /// <param name="participant"></param>
+        /// <param name="days"></param>
+        /// <returns></returns>
         Task SendMultiDayHearingNotificationAsync(HearingDto hearing, IList<ParticipantDto> participants, int days);
+        
+        
+        
+        
+        
+        
+        
+        
+        // Task SendNewUserAccountNotificationAsync(Guid hearingId, ParticipantDto participant, string password);
```

Comment:
could we remove this commented out code?

Reply:
done